content/docs/reference/design-docs/timetable.en.md

```diff
@@ -86,21 +127,44 @@ path:
  - {id: c, deleted: true, waypoint: true, trigram: ABC}
  - {id: d, waypoint: true, track: toto, offset: 42}
 
-engineering_margins:
- - {from: b, to: c, value: "2%", distribution: MARECO}
+stops:
+ - {at: a, for: PT5M}
 
-standard_margins:
-  # the begin and end waypoints are always implicitly added
+# there are two mutually exclusive ways to add time constraints:
+# margins and arrival times. Both methods share parameters, such
+# as the distribution, intermediate waypoints, and transitions.
+time_constraints:
+  # algorithm used to distribute time over space
   distribution: MARECO
-  intermediate_waypoints: [b]
-  values: ["5%", "2%"]
-
-# all durations and times are specified using ISO 8601
-scheduled_points:
- - {at: a, arrival: PT0M, stop_for: PT5M}
- - {at: b, arrival: PT10M}
- - {at: c, arrival: PT25M}
 
+  # time constraints apply on the entire path. intermediate
+  # waypoints delimit zones which can be configured with distinct
+  # time constraints.
+  intermediate_waypoints: [b, c]
+
+  # can be BEFORE, AFTER or AUTO. If AUTO, the transition type is
+  # unspecified, and left for the system to figure out.
+  # There has to be as many transitions as intermediate points.
+  # If the field is omited, the default transition type is AUTO.
+  transition_types: ["AFTER", "AUTO"]
```

Comment:
It feels like it will never be specified by the user, and so never used.

Reply:
It's the only difference between construction margins and baseline margins

Reply:
this is gone, as there are no more construction margins